core/shark_turbine/kernel/_support/tracing.py

```diff
@@ -404,9 +412,23 @@ def current() -> "LaunchContext":
             return DebugLaunchContext()
 
     def __enter__(self) -> "LaunchContext":
+        # Push an indexing context with the constand bindings for this launch
+        # context in it.
+        # TODO: Is this the most pythonic way to do this? Ask Stella before
```

Comment:
Could use a look here. The LaunchContext by itself now also generates an IndexingContext.

Reply:
I think this is fine.